tutorials/s4sdk-cloud-foundry-sample-application/s4sdk-cloud-foundry-sample-application.md

```diff
@@ -198,45 +186,51 @@ This tells Maven to remove any files from previous assemblies (clean) and to ass
 Now the previously mentioned `manifest.yml` comes into play – it's the deployment descriptor used by `Cloud Foundry`.
 
 
-```
-  ---
-  applications:
+```yaml
+
+---
+applications:
 
 - name: firstapp
-  memory: 768M
-  host: firstapp-D123456
-  # random-route: true # used instead of "host"
+  memory: 1024M
+  timeout: 300
+  routes:
+    - route: firstapp-<SUBACCOUNT>.cfapps.sap.hana.ondemand.com
```

Comment:
That's not the content as generated, is it? I suggest keeping `random-route: true` for now. Already trying to prepare for security step here makes this too difficult (would need explanation of where to get the subdomain from etc). Put that into the security tutorial.

Reply:
host is actually deprecated and will be rejected when trying to push with CF. Also this makes it easier to test subsequent tutorials. I am not sure where we would have to adapt the following tutorials in order to account for this. That's why i'd like to keep it this way.

Reply:
updated